ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,21 @@
+ages = [29 , 19 , 17 , 26 , 32 , 16 , 21]
+print(ages)
+print(ages[4])
+print(ages[-2])
+print(ages[2:6])
+print(ages[:4])
+print("17" in ages)
+print("42" in ages)
+ages[2] = "18"
+print(ages)
```

Comment:
The third value in `ages` should be overwritten by `18`, not `"18"`:

---

![image](https://user-images.githubusercontent.com/16143771/149632104-e3bb678b-3e39-497e-bd6d-ef03a5a70202.png)